app/src/main/res/layout/fragment_logged_in.xml

```diff
@@ -11,27 +11,28 @@
         android:layout_height="wrap_content"
         android:layout_gravity="center" />
 
-    <FrameLayout
-        android:id="@+id/signed_panel"
+    <LinearLayout
+        android:orientation="horizontal"
         android:layout_width="match_parent"
-        android:layout_height="match_parent"
-        android:visibility="gone">
```

Comment:
Why remove the visibility attribute?

Reply:
Good catch.  This should not be in here.

Reply:
Fixed.